### PR TITLE
Ingress doc clarifications

### PR DESCRIPTION
**Pull request by bprashanth**
Last minute doc clarifications. Assigning to Jeff since he has context, feel free to re-assign.

### CODE REVIEW

File: cluster/addons/cluster-loadbalancing/glbc/README.md [hunk 1]
```diff
@@ -6,6 +6,15 @@ This cluster addon is composed of:
 
 It relies on the [Ingress resource](../../../../docs/user-guide/ingress.md) only available in Kubernetes version 1.1 and beyond.
 
+## Prerequisites
+
+Before you can receiver traffic through the GCE L7 Loadbalancer Controller you need:
+* A Working Kubernetes 1.1 cluster
+* At least 1 Kubernets [NodePort service](../../../../docs/user-guide/services.md#type-nodeport) (this is the endpoint for your Ingress)
```
Comment (jlowdermilk) [inline cluster/addons/cluster-loadbalancing/glbc/README.md]: s/Kubernets/Kubernetes

**Comment by jlowdermilk** [comment]
lgtm, thanks

**Comment by bprashanth** [comment]
Added a line about HTTPS, re-adding lgtm to keep merge bot happy

**Comment by bprashanth** [comment]
@k8s-bot test this please

**Comment by bprashanth** [comment]
@k8s-bot test this please

**Comment by bprashanth** [comment]
Jenkins flakes are unrelated, shippable and unittests passed, merging manually for release

**Pull request merged**